apps/docs/src/.env.example

```diff
@@ -0,0 +1,2 @@
+# move to apps/docs/src/.env.local file
```

Comment:
```suggestion
# move to .env.local file, e.g. by running:
# cp .env.example .env.local
```

Reply:
```
# move to apps/docs/src/.env.local file, e.g. by running:
# cp apps/docs/src/.env.example apps/docs/src/.env.local
```

or
```
# move to src/.env.local file, e.g. by running:
# cp src/.env.example src/.env.local
```

-> usually, you would run scripts from the root folder of the repo / app, right?

Reply:
The root of the app yes :D Your second suggestion sound good!